src/HydraAuction/Runner.hs

```diff
@@ -172,6 +173,14 @@ executeTestRunner runner = do
           (MkExecutionContext {tracer = tracer, node = node, actor = Alice})
           runner
 
+executeDockerRunner :: Runner () -> IO ()
+executeDockerRunner runner = do
+  runProcess_ "make start-docker"
```

Comment:
I think it better should stop `docker-compose` after. And also we could reuse same code for both docker cases.